I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script/script.rpy", line 36, in script
    $ renpy.pause(5.0, hard=true)
  File "game/script/script.rpy", line 36, in <module>
    $ renpy.pause(5.0, hard=true)
NameError: name 'true' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script/script.rpy", line 36, in script
    $ renpy.pause(5.0, hard=true)
  File "D:\Programs\renpy-7.3.5-sdk\renpy\ast.py", line 914, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\Programs\renpy-7.3.5-sdk\renpy\python.py", line 2028, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script/script.rpy", line 36, in <module>
    $ renpy.pause(5.0, hard=true)
NameError: name 'true' is not defined

Windows-8-6.2.9200
Ren'Py 7.3.5.606
There is no god in Hinterland 1.0
Sat Sep 03 16:09:20 2022
